### PR TITLE
Story6 task1

### PR DESCRIPTION
**Pull request by emferrari31**
Story6 bug fixed

### CODE REVIEW

**Review by joseph-padfield — COMMENTED** [review]
Looks good :)

File: README.md [hunk 1]
```diff
@@ -276,14 +276,59 @@ GET
...
   - If the data is retrieved successfully it will return an array with the courses scheduled
+  
+  
+**/getEvents**
+GET
+-  Gets all the future events present in database.
+-  Required: 
+-  Optional: - ?categoryValue=[integer]
+             - ?searchTerm=[alphanumeric]
+			 - ?past=[tinyInt]  --- if 1, returns past events
```
Comment (matt-ioa) [inline README.md:287]: Can you reindent this so it's in line with the options above it?

File: README.md [hunk 2]
```diff
@@ -276,14 +276,59 @@ GET
...
+
+HT - 23/9/24
+
+**/getGender**
+GET
+
+- Retrieves all the gender information from the gender table on the academy portal database. 
+- Data Format: 
+  - `{"id":1,"gender":"Male"},{"id":2,"gender":"Female"},{"id":3,"gender":"Non-binary"},
+        {"id":4,"gender":"Prefer not to say"}]}}`
```
Comment (matt-ioa) [inline README.md:320]: It looks like there's an extra square bracket and a couple of extra curly brackets here: is this the actual data format needed, or is it something else (like an indexed array of associative arrays)?

File: src/ViewHelpers/StagesPageViewHelper.php [hunk 3]
```diff
@@ -95,7 +95,7 @@ public static function displayStages(array $stages): string
                 $result .= '<td class="col-xs-2 text-center"><a class="toggleEditForm">Edit</a></td>';
                 if (empty($stage->getOptions()) && $stage->getHasAssignees() === 0) {
                     $result .= '<td class="col-xs-2 text-center"><a data-hasOptions="0" data-id="'
-                        . $stage->getStageId() . '" class="text-danger delete disabled">Delete</a></td>';
+                        . $stage->getStageId() . '" class="text-danger delete">Delete</a></td>';
```
Comment (matt-ioa) [inline src/ViewHelpers/StagesPageViewHelper.php:98]: This code is all fine BUT your pull request is doing two completely different things here: the documentation of some routes and also, unrelatedly, fixing the issue with the delete button on the application stages page.

Ideally, these would be two separate pull requests, but I'm happy to merge them together. Just be aware not to clump together unrelated things in the future!

**Review by matt-ioa — CHANGES_REQUESTED** [review]
One small change and a question or two to answer

**Review by matt-ioa — APPROVED** [review]

**Pull request merged**